exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/TransactionResult.java

```diff
@@ -114,10 +106,10 @@ private static TransactionResult valueOf(
   public abstract OptionalInt getErrorCode();
 
   /**
-   * Returns an optional description of a transaction if its execution resulted in an error.
-   * @return a description of an error, or {@code Optional.empty()} if there is no description
+   * Returns the description of a transaction if its execution resulted in an error;
+   * or an empty string if there is no description. Never {@code null}.
```

Comment:
I think it's redundant here to explicitly state that it's never `null`. If method is not annotated as `@Nullable`, user should be confident that he won't get `null` from it.